modules/ROOT/partials/integrations/svelte-quick-start.adoc

```diff
@@ -119,7 +118,7 @@ export default defineConfig({
 ----
 . Open `+src/App.svelte+` and replace the contents with:
 +
-[source,html]
+[source,javascript]
```

Comment:
```suggestion
[source,svelte]
```

According to google, you can provide svelte syntax

Reply:
Updated, will need to confirm it works when rendered.